concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetAnonymityRevokers.java

```diff
@@ -3,42 +3,43 @@
 import com.concordium.sdk.ClientV2;
 import com.concordium.sdk.Connection;
 import com.concordium.sdk.Credentials;
+import com.concordium.sdk.exceptions.ClientInitializationException;
 import com.concordium.sdk.requests.BlockHashInput;
-import lombok.SneakyThrows;
-import lombok.val;
 import picocli.CommandLine;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
 
+import java.net.MalformedURLException;
 import java.net.URL;
-
+import java.util.concurrent.Callable;
 
 @Command(name = "GetAnonymityRevokers", mixinStandardHelpOptions = true)
-public class GetAnonymityRevokers implements Runnable {
+public class GetAnonymityRevokers implements Callable<Integer> {
     @Option(
             names = {"--endpoint"},
             description = "GRPC interface of the node.",
             defaultValue = "http://localhost:20001")
     private String endpoint;
 
-
-    @SneakyThrows
     @Override
-    public void run() {
+    public Integer call() throws ClientInitializationException, MalformedURLException {
```

Comment:
Why not just `Runnable`?
Can it ever return some other value than `0`?

Reply:
Call here is not being used to specify a return code. Returning an exit code is a side effect which might be useful in further development. 

Currently its being used so that Checked Exceptions can be added to the Method signature. Since earlier approach to use `@SneakyThrows` in the Examples code was not an Elegant Solution. 
The Library used to create console application automatically handles the Exceptions and prints them to the console.

Reply:
@EmilLa1  - please do suggest if we need to revert back to the previous approach.

Reply:
Well, I don't see why one would care about checked exceptions in this particular cli example. And as it is just an example, then I would prefer it to be as simple as possible. 

But I'll let you decide.

Reply:
Thanks @EmilLa1

The Only option barring using `@SneakyThrows` is to use a `Callable` instead of a `Runnable` and handle the errors.
We should be able to use the return code also in later examples as a way to provide return code when needed for a Runtime exception. 
Allow us to keep it as it is because it provides better and more read-able code structure.